README.md

```diff
@@ -1,3 +1,11 @@
 # Google integration in Nextcloud
 
 🇬 Google integration allows you to automatically import your Google calendars, contacts, photos and files into Nextcloud.
+
+## 🚀 Installation
+
+In your Nextcloud, simply enable the Google Integration app through the Apps management. The Google Integration app is only included in Nextcloud v20 and higher.
+
+## 🔧 Setup
+
+The app needs some setup in the Google API Console in order to work. To do this, go to Settings > Administration > Connected accounts and follow the instructions under "Google integration".
```

Comment:
It's not clear that your first talk about the google side and then the Nextcloud settings.

```suggestion
The app needs some setup in the Google API Console in order to work. To do this, go to Nextcloud Settings > Administration > Connected accounts and follow the instructions of "Google integration" section.
```